Gemfile.lock

```diff
@@ -93,7 +93,9 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.5)
+    mimemagic (0.3.10)
+      nokogiri (~> 1)
+      rake
```

Comment:
The application won't start locally, because of:
```
Your bundle is locked to mimemagic (0.3.5) from rubygems repository https://rubygems.org/ or installed locally, but that version can no longer be found in that source.
That means the author of mimemagic (0.3.5) has removed it. You'll need to update your bundle to a version other than mimemagic (0.3.5) that hasn't been removed in order to install.
```